roofit/roofitcore/test/test_lib.h

```diff
@@ -0,0 +1,185 @@
+/*****************************************************************************
+ * Project: RooFit                                                           *
+ * Package: RooFitCore                                                       *
+ * @(#)root/roofitcore:$Id$
+ * Authors:                                                                  *
+ *   PB, Patrick Bos,     NL eScience Center, p.bos@esciencecenter.nl        *
+ *                                                                           *
+ * Redistribution and use in source and binary forms,                        *
+ * with or without modification, are permitted according to the terms        *
+ * listed in LICENSE (http://roofit.sourceforge.net/license.txt)             *
+ *****************************************************************************/
+
+#ifndef ROOT_TEST_LIB_H
+#define ROOT_TEST_LIB_H
+
+#include <sstream>
+
+#include <memory>  // make_unique
+
+#include "RooWorkspace.h"
+#include "RooRandom.h"
+#include "RooAddPdf.h"
+#include "RooDataSet.h"
+#include "RooRealVar.h" // for the dynamic cast to have a complete type
```

Comment:
In general
1. RooFit
2. Root
3. <std>

would be the safer (=more strict, would point out forgotten includes in those headers) order, but that's mostly relevant for headers.